frontend/app/.server/domain/entities/benefit-renewal.entity.ts

```diff
@@ -20,26 +20,26 @@ export type BenefitRenewalRequestEntity = Readonly<{
       }>;
       PersonContactInformation: ReadonlyArray<
         Readonly<{
-          Address: ReadonlyArray<
+          Address?: ReadonlyArray<
```

Comment:
Should this Entity have an optional `Address` (same with the fields that were changed below and the DTO)? 

When a renewal application is submitted, the request to the downstream API should always have an address set. If the user didn't fill out the address information form, the application would grab the address on file for the client and send that instead.